fedimint-bin-tests/src/federation.rs

```diff
@@ -185,7 +185,7 @@ pub async fn run_dkg(root_task_group: &TaskGroup, servers: usize) -> anyhow::Res
     async fn create_tls(id: usize, sender: Sender<String>) -> anyhow::Result<()> {
         // set env vars
         let bin_dir = env::var("FM_BIN_DIR")?;
-        let server_name = format!("Server-{id}");
+        let server_name = format!("Server {id}!");
```

Comment:
I wonder if we should just

```
 info!(peer-id=peer_id, out-dir=out_dir", "creating TLS certs created for peer in {out_dir}");
```

kind of thing, instead of interpolating strings for logging.

Reply:
Isn't this being used as a cmd line arg, not logging?

Reply:
I don't understand that nit either, looks good to me, just test code running the dkg.